test/fixtures/poro.rb

```diff
@@ -38,9 +38,16 @@ class ProfilePreviewSerializer < ActiveModel::Serializer
 
 Post     = Class.new(Model)
 Like     = Class.new(Model)
-Author   = Class.new(Model)
+Author   = Class.new(Model) do
+  # To confirm cache_key is set using updated_at and cache_key option passed to cache
+  undef_method :cache_key
+end
 Bio      = Class.new(Model)
 Blog     = Class.new(Model)
+Article  = Class.new(Model) do
+  # To confirm error is raised when cache_key is not set and cache_key option not passed to cache
+  undef_method :cache_key
+end
```

Comment:
maybe we can subclass this in the test as

```ruby
class UncachedArticle < Article
  undef_method :cache_key
end
```

so the fixture here doesn't need to change? etc.

Also, the ArticleSerializer below can be defined in the test, rather than adding a new one here